internal/cmd/commands/scopescmd/rotate_keys.go

```diff
@@ -0,0 +1,117 @@
+package scopescmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/boundary/api"
+	"github.com/hashicorp/boundary/api/scopes"
+	"github.com/hashicorp/boundary/internal/cmd/base"
+	"github.com/mitchellh/cli"
+	"github.com/mitchellh/go-wordwrap"
+	"github.com/posener/complete"
+)
+
+var (
+	_ cli.Command             = (*RotateKeysCommand)(nil)
+	_ cli.CommandAutocomplete = (*RotateKeysCommand)(nil)
+)
+
+type RotateKeysCommand struct {
+	*base.Command
+}
+
+func (c *RotateKeysCommand) Synopsis() string {
+	return wordwrap.WrapString("Rotate the keys within a scope", base.TermWidth)
+}
+
+func (c *RotateKeysCommand) Help() string {
+	return base.WrapForHelpText([]string{
+		"Usage: boundary scopes rotate-keys [args]",
+		"",
+		"  Generate a new root key and new DEKs for a scope. Example:",
```

Comment:
I wonder if we should just double down on the `KEK` terminology as well
```suggestion
		"  Generate a new KEK and new DEKs for a scope. Example:",
```
What do you think?